lib/Db/ImageGenerationMapper.php

```diff
@@ -163,6 +164,11 @@ public function cleanupGenerations(int $maxAge = Application::MAX_GENERATION_IDL
 
 		/** @var ImageGeneration[] $generations */
 		$generations = $this->findEntities($qb);
+		$qb->resetQueryParts();
```

Comment:
I usually create a new qb via `$qb = $this->db->getQueryBuilder();`, feels cleaner somehow

Reply:
Thanks! Done!